tests/UnifiedSpecTests/UnifiedTestRunner.php

```diff
@@ -60,26 +61,20 @@ final class UnifiedTestRunner
      * however, syntax from 1.9, 1.10, and 1.11 has not been implemented. */
     public const MAX_SCHEMA_VERSION = '1.12';
 
-    /** @var MongoDB\Client */
-    private $internalClient;
+    private Client $internalClient;
 
-    /** @var string */
-    private $internalClientUri;
+    private string $internalClientUri;
 
-    /** @var bool */
-    private $allowKillAllSessions = true;
+    private bool $allowKillAllSessions = true;
 
-    /** @var EntityMap */
-    private $entityMap;
+    private ?EntityMap $entityMap = null;
 
     /** @var callable(EntityMap):void */
     private $entityMapObserver;
```

Comment:
This property is never set. https://github.com/mongodb/mongo-php-library/pull/1129

Reply:
Used by Atlas Testing. See https://github.com/mongodb/mongo-php-library/pull/1129#discussion_r1262872050.